source/supported-hardware/qsp.rst

```diff
@@ -0,0 +1,70 @@
+.. _getting-started_qsp:
+
+
+QSP Simics Platform
+-----------------------
+
+
+Building SBL
+^^^^^^^^^^^^
+
+To build |SPN| for Intel QSP simics platform (it is the same image as QEMU, SBL will auto detect the platform at runtime)::
+
+    python BuildLoader.py build qemu
+
+The output images are generated under ``Outputs/qemu/SlimBootloader.bin`` directory.
+
+.. note:: QSP simics platform does not require stitching the |SPN| into an IFWI.
+
+
+
+Setting up QSP Simics
+^^^^^^^^^^^^^^^^^^^^^
+
+Please head over to this link to setup Intel Simics® Simulator Public Release Preview:
+
+|https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/|
```

Comment:
Use this link - https://software.intel.com/content/www/us/en/develop/articles/simics-simulator.html